drivers/hmis/app/models/hmis/hud/custom_client_address.rb

```diff
@@ -68,4 +74,17 @@ def self.use_values
   def self.type_values
     TYPE_VALUES
   end
+
+  def validate_required_fields?
+    move_in_type?
+  end
+
+  # maybe there's a list of states somewhere else we can use?
+  USA_STATES = ['AK', 'AL', 'AR', 'AS', 'AZ', 'CA', 'CO', 'CT', 'DE', 'FL', 'GA', 'GU', 'HI', 'IA', 'ID', 'IL', 'IN', 'KS', 'KY', 'LA', 'MA', 'MD', 'ME', 'MI', 'MN', 'MO', 'MP', 'MS', 'MT', 'NC', 'ND', 'NE', 'NH', 'NJ', 'NM', 'NV', 'NY', 'OH', 'OK', 'OR', 'PA', 'PR', 'RI', 'SC', 'SD', 'TN', 'TX', 'UT', 'VA', 'VI', 'VT', 'WA', 'WI', 'WV', 'WY'].freeze
```

Comment:
Huh, I would have thought we'd have had this somewhere, but I don't see it.  I wonder if something like https://github.com/thecodecrate/city-state would be useful.  Overkill here, and we have a bunch of geocode data, so you should be able to do something like: 
`GrdaWarehouse::Shape::State.pluck(:stusps).sort` but that's also dependent on having loaded all the state shapes, which should have happened in production.

Reply:
We have it for HMIS in `drivers/hmis/lib/pick_list_data/states.json`, the `STATE` picklist uses it. So you should be able to re use that!

Reply:
Thanks! I think I might leave it as-is if that's okay. I would prefer for this to be non-dry and self-contained rather than add complicated stuff inside a class definition (such as db queries or reaching into graphql type classes). Class loader bugs can be really hard to unwind imo.